### ISSUE TITLE
The doc of Sobol' indices has issues

### ISSUE THREAD

**Issue by mbaudin47** [openturns/openturns#2332]
The doc of Sobol' indices has issues:0

https://openturns.github.io/openturns/latest/theory/reliability_sensitivity/sensitivity_sobol.html#aggregated-sobol-indices

The index k is an index in the variance of Vi and an exponent in Var :

![image](https://github.com/openturns/openturns/assets/31351465/65d0a4e6-8ed1-4801-ae34-e61d726c8255)

**Comment by adutfoy** [comment]
I agree there is a typo for the component $Y^{(k)}$ which must be written as $Y_k$ to be coherent with the begining of the file.

But I think we have no other way to indicate that the conditional variance given $X_i$  is calculated on $Y_k$: the subindices of $V$ as well as $T$ are already used to indicate the index of the conditioning variable $X_i$. I propose to keep on $V_i^{(k)}$ and $T_i^{(k)}$.

I modified the documentation in PR#2574.

@mbaudin47 : do you agree?

**Comment by mbaudin47** [comment]
> @mbaudin47 : do you agree?

I agree.
